scania-digital-design-system/sdds-website
# fix: dependabot upgrades

**Describe pull-request**  
Bumping up dependabot reported packages (see in list of PRs)


**How to test**  
Just try to run code as usual and see if it builds and website is available on localhost.
